iroh-resolver/src/chunker.rs

```diff
@@ -22,6 +22,7 @@ impl Chunker {
     }
 
     pub fn fixed_with_size(size: usize) -> Self {
+        assert!(size > 0 && size <= DEFAULT_CHUNK_SIZE_LIMIT);
```

Comment:
Size could be larger, no need to restrict it to the default

Reply:
you really don't want to create chunks that you can't bitswap. We had this at actyx once, and it was not fun to figure out what was wrong. But I guess that the chunker is independent of such concerns, so OK I will remove.

Reply:
yeah, we should have some limit probably for sanity reasons, but it could be much larger than 256kib